pcdsdevices/attenuator.py

```diff
@@ -125,6 +127,18 @@ class AttBase(FltMvInterface, PVPositioner):
     # Aux Signals
     calcpend = Cpt(EpicsSignalRO, ':COM:CALCP', kind='omitted')
 
+    # 3rd harmonic frequency components
```

Comment:
Added the 3rd harmonic frequency components to the `AttBase` class  and gotten rid of AttBase3rd- is this kind of what you meant @ZLLentz  (***This might require some reworking of the Att classes so that we don't have separate 1st and 3rd harmonic devices***)

Or did I go a bit too far here with this approach?

Reply:
I think this approach makes a lot of sense, but you've actually added PVs to FeeAtt that don't exist- it doesn't have a 3rd harmonic calculator.

Reply:
Ohh noo - i see, thanks for pointing this out. I'm going to do some thinking here...

Reply:
I think I'm gonna have to go back to the original implementation of having two base classes here?, one for 3rd harmonics and one for the fundamental one?  
Or I can try to have `FeeAtt` not inherit from `AttBase` and implement its own thing... 
Or maybe other better suggestions?

Reply:
Maybe just add an `AttBaseWith3rd` intermediate class?

Reply:
And have everybody inherit this class except `FeeAtt`? is that what you mean?

Reply:
Yes, that's what I'm saying.

The other approach would be to orphan the `FeeAtt` class onto its own island which is also fine- we're not revisiting that device ever (and in ops you're not "supposed" to use it)